unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/PrivateChatWindowComponentView.cs

```diff
@@ -82,9 +84,10 @@ public override void Dispose()
         if (!gameObject) return;
 
         if (userContextMenu != null)
-        {
             userContextMenu.OnBlock -= HandleBlockFromContextMenu;
-        }
+
+        if (chatView!= null)
+            chatView.OnUnblockUser += HandleUnblockFromButtonInChat;
```

Comment:
Be careful because you're subscribing this twice: here and in the `Awake()`. Is this intentional?

Reply:
Yep, very silly mistake.